<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdministradorPanel</class>
 <widget class="QMainWindow" name="AdministradorPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2136</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel del administrador</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>501</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Panel del administrador</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGestionarProveedores">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gestionar proveedores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGestionarPedidosProveedores">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>410</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gestionar pedidos a proveedores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalirAplicacion">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>560</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salir de la aplicacion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPedidos">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>410</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ver pedidos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGestionarRepuestos">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>260</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gestionar repuestos</string>
    </property>
   </widget>
   <widget class="QWidget" name="imagen" native="true">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>30</y>
      <width>411</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2136</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
